promptsource/templates/movie_rationales/templates.yaml

```diff
@@ -58,29 +63,32 @@ templates:
 
       - {{ evidences | join("\n- ") }}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
+      choices_in_prompt: false
+      metrics:
+      - Other
+      original_task: false
     name: Generate evidences
     reference: From the review, extract the spans of text that let us think that the
       review is positive or negative.
   f11ea73a-3a03-43d8-90d8-4da3905161c2: !Template
-    answer_choices: null
+    answer_choices: Negative ||| Positive
     id: f11ea73a-3a03-43d8-90d8-4da3905161c2
     jinja: 'Review: {{review}}
 
       ====
 
-      Is this review negative or positive? Extract from the review the passage that
-      proves this choice. |||
+      Is this review {{answer_choices[0]}} or {{answer_choices[0]}}? Extract from
+      the review the passage that proves this choice. |||
 
-      {{["Negative", "Positive"][label]}}
+      {{answer_choices[label]}}
 
       - {{ evidences | join("\n- ") }}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
```

Comment:
```suggestion
```

Reply:
Taking this out because Accuracy means "exact string match" but this requires parsing.